docs/index.asciidoc

```diff
@@ -43,14 +44,84 @@ filter plugins.
 
 &nbsp;
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured geo data added at root level
+** `v1`: uses fields that are compatible with Elastic Common Schema (for example, `[client][geo][country_name]`)
```

Comment:
do not think we want the `[client][geo][country_name]` example here, no example would do fine

Reply:
oh, my bad to copy and paste it